CMakeLists.txt

```diff
@@ -1,25 +1,22 @@
 cmake_minimum_required(VERSION 3.4 FATAL_ERROR)
-PROJECT(sioclient)
+PROJECT(sioclient
+        VERSION 3.1.0
+        )
 
 option(BUILD_SHARED_LIBS "Build the shared library" OFF)
-option(BUILD_UNIT_TESTS  "Builds unit tests target" OFF)
+option(BUILD_UNIT_TESTS "Builds unit tests target" OFF)
 
-set(MAJOR 1)
-set(MINOR 6)
-set(PATCH 0)
-
```

Comment:
Do not remove these lines as, `MAJOR`, `MINOR` and `PATCH` are being used as a CMake property